### PR TITLE
[ja] Change paths of debug pages

### LINKED ISSUE #38797: [ja] Some links to debug page are outdated
**This is a Feature Request**

**What would you like to be added**
PR #34162 changed the location of some .md files.
We need to update the links to those pages.

**Why is this needed**
Because the links are broken and the pages cannot be referenced.
Also, we can change some links to English page to Japanese page.

I checked "/docs/tasks/debug" by using grep command.
There are many links that need to be updated.
I would like to divide them into three cases, and create each PRs for easier to review.

- Only add "/ja"
  - e.g.: /docs/tasks/debug/debug-cluster/audit/ -> /ja/docs/tasks/debug/debug-cluster/audit/
- Change path
  - e.g.: /ja/docs/tasks/debug-application-cluster/audit/ -> /ja/docs/tasks/debug/debug-cluster/audit/
- Change path and add "/ja"
  - e.g.: /docs/tasks/debug-application-cluster/resource-usage-monitoring/ -> /ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring/

**Comments**
/language ja
/assign

### PR DESCRIPTION
**Pull request by shogohida**
Signed-off-by: Shogo Hida <shogo.hida@gmail.com>

Fixes #38797 

This PR changes the paths of debug pages. The example shown on the issue is below.

- Change path
  - e.g.: /ja/docs/tasks/debug-application-cluster/audit/ -> /ja/docs/tasks/debug/debug-cluster/audit/
/ja/docs/tasks/debug-application-cluster

### CODE REVIEW

File: content/ja/docs/tasks/run-application/force-delete-stateful-set-pod.md [hunk 1]
```diff
@@ -74,5 +74,4 @@ StatefulSet Podの強制削除は、常に慎重に、関連するリスクを
 ## {{% heading "whatsnext" %}}
 
 
-[StatefulSetのデバッグ](/ja/docs/tasks/debug-application-cluster/debug-stateful-set/)の詳細
-
+[StatefulSetのデバッグ](/ja/docs/tasks/debug/debug-application/debug-stateful-set/)の詳細
```
Comment (atoato88) [inline content/ja/docs/tasks/run-application/force-delete-stateful-set-pod.md:77]:
```suggestion
[StatefulSetのデバッグ](/ja/docs/tasks/debug/debug-application/debug-statefulset/)の詳細
```

**Review by atoato88 — COMMENTED** [review]
Please check comment.

**Comment by atoato88** [comment]
First of all, thank you for creating this PR!
Almost all are OK for me.

But I `grep` "/debug-application-cluster/" on `ja` content and found other target files followings:
1. ja/docs/concepts/cluster-administration/cluster-administration-overview.md
1. ja/docs/concepts/cluster-administration/manage-deployment.md
1. ja/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md
1. ja/docs/tasks/debug/debug-cluster/local-debugging.md

Files 1. to 3. have the link to "/debug-application-cluster/" and the URL has `ja` page. so we can update those, I think.

@t-inu 
Last file 4. have also the link to "/debug-application-cluster/" but the URL doen't have `ja` page.
(And we should update `task` with `tasks` in URL on this page...)
What do we do about this?

@shogohida @t-inu 
Or those files are out of scope on this PR?

**Comment by shogohida** [comment]
@atoato88 
Thanks for reviewing my PR! I think the files you mentioned are out of the scope of this PR because [the issue](https://github.com/kubernetes/website/issues/38797) is based on debug files as the title says and I just changed paths which already have /ja. 

The issue shows these 3 cases and I just took the second case for this PR. The requirement is to create each PR for each case.

- Only add "/ja"
  - e.g.: /docs/tasks/debug/debug-cluster/audit/ -> /ja/docs/tasks/debug/debug-cluster/audit/
- Change path
  - e.g.: /ja/docs/tasks/debug-application-cluster/audit/ -> /ja/docs/tasks/debug/debug-cluster/audit/
- Change path and add "/ja"
  - e.g.: /docs/tasks/debug-application-cluster/resource-usage-monitoring/ -> /ja/docs/tasks/debug/debug-cluster/resource-usage-monitoring/

**Comment by atoato88** [comment]
/lgtm

Thank you for quick update and detailed comment! 👍

**Comment by shogohida** [comment]
@inductor 
It seems I assigned you by accident so please ignore this PR. This is my first issue and I made a mistake to assign an approver :pray:

**Review by t-inu — APPROVED** [review]
I reviewed and LGTM. Thanks!

**Comment by shogohida** [comment]
@t-inu 
Thanks for your review! I will solve other cases you mentioned in the issue after this PR!

**Pull request merged**